tempus_fugit_minecraft/window.py

```diff
@@ -324,8 +324,8 @@ def center_labels(self, width: int, height: int) -> None:
         self.resume_label.y = height // 2 - 45
         self.quit_label.y = height // 2 - 90
 
+    """ Issue 68 Configure OpenGL to draw in 2d."""
     def set_2d(self) -> None:
-        """Configure OpenGL to draw in 2d."""
```

Comment:
Docstrings go inside the functions, not above them. See here: [PEP-257](https://peps.python.org/pep-0257/)

This applies to all other similar changes, such as for `set_3d`, `on_draw`, `draw_pause_menu`, `draw_focused_block`, `draw_reticle`, `decrease_speed`, and `increase_speed`.

Reply:
Done